### PR TITLE
Sound mixer part 1

### PR DESCRIPTION
**Pull request by grungussuss**
## About The Pull Request
image
## Why It's Good For The Game
gives players more control over how loud they want certain sounds to be
## Changelog
:cl: grungussuss
sound: the volume that ship ambience, ambience, radio noise and announcements play at can now be tweaked in preferences, check your preferences!
/:cl:

### CODE REVIEW

**Comment by StrangeWeirdKitten** [comment]
Absolutely fucking based

File: code/modules/client/preferences/sounds.dm [hunk 1]
```diff
@@ -1,10 +1,17 @@
...
 
-/datum/preference/toggle/sound_ambience/apply_to_client(client/client, value)
+	minimum = 0
+	maximum = 200
+
+/// default value is max/2 because 100 1x modifier, while 200 is 2x
+/datum/preference/numeric/sound_ambience_volume/create_default_value()
+	return maximum/2
+
+/datum/preference/toggle/sound_ambience_volume/apply_to_client(client/client, value)
```
Comment (Ghommie) [inline code/modules/client/preferences/sounds.dm:14]:
```suggestion
/datum/preference/numeric/sound_ambience_volume/apply_to_client(client/client, value)
```

File: code/modules/client/preferences/sounds.dm [hunk 2]
```diff
@@ -85,12 +93,19 @@
...
 
-/datum/preference/toggle/sound_lobby/apply_to_client_updated(client/client, value)
+	minimum = 0
+	maximum = 200
+
+/// default value is max/2 because 100 1x modifier, while 200 is 2x
+/datum/preference/numeric/sound_lobby_volume/create_default_value()
+	return maximum/2
+
+/datum/preference/toggle/sound_lobby_volume/apply_to_client_updated(client/client, value)
```
Comment (Ghommie) [inline code/modules/client/preferences/sounds.dm:108]:
```suggestion
/datum/preference/numeric/sound_lobby_volume/apply_to_client_updated(client/client, value)
```

File: code/modules/client/preferences/sounds.dm [hunk 3]
```diff
@@ -102,13 +117,20 @@
...
 
-/datum/preference/toggle/sound_ship_ambience/apply_to_client_updated(client/client, value)
+	minimum = 0
+	maximum = 200
+
+/// default value is max/2 because 100 1x modifier, while 200 is 2x
+/datum/preference/numeric/sound_ship_ambience_volume/create_default_value()
+	return maximum/2
+
+/datum/preference/toggle/sound_ship_ambience_volume/apply_to_client_updated(client/client, value)
```
Comment (Ghommie) [inline code/modules/client/preferences/sounds.dm:133]:
```suggestion
/datum/preference/numeric/sound_ship_ambience_volume/apply_to_client_updated(client/client, value)
```

**Review by Ghommie — CHANGES_REQUESTED** [review]

**Comment by W4rd3nn** [comment]
Just gonna bring this up, but could you also do a volume setting for announcements? like the "meteors have been detected" kind. they can be pretty grating but turning them off completely also causes you to miss them.

**Comment by Ghommie** [comment]
What gives?

**Pull request closed without merging**